### PR TITLE
Add new Test 

### PR DESCRIPTION
**Pull request by HynRidge**
Add unit testing for class tagNameContainsCharPredicate

### CODE REVIEW

**Review by Ma-Yueran — APPROVED** [review]
Looks good to me, but I'm not familiar with this class, so I'm not 100% sure.

File: src/test/java/seedu/address/model/tag/TagNameContainsCharPredicateTest.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+                firstPredicateKeywordList);
+        assertTrue(firstPredicate.equals(firstPredicateCopy));
+
+        // different types -> returns false
+        assertFalse(firstPredicate.equals(1));
+
+        // null -> returns false
+        assertFalse(firstPredicate.equals(null));
+
+        // different person -> returns false
```
Comment (luo-git) [inline src/test/java/seedu/address/model/tag/TagNameContainsCharPredicateTest.java]: "Person" should be predicate here?

File: src/test/java/seedu/address/model/tag/TagNameContainsCharPredicateTest.java [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+        //Words end with different suffix
+        assertTrue(firstPredicate.test(new TagBuilder().withTagName("test123Hello").build()));
+
+        //Words contain keyword in the middle
+        assertTrue(firstPredicate.test(new TagBuilder().withTagName("HellotestWorld").build()));
+    }
+
+    @Test
+    public void test_tagNameContainsKeyword_returnsFalse() {
```
Comment (luo-git) [inline src/test/java/seedu/address/model/tag/TagNameContainsCharPredicateTest.java]: Method name should be `test_keywordMissing_returnsFalse`?

**Review by luo-git — CHANGES_REQUESTED** [review]
Thanks for adding tests! =)
Just need some minor refactoring.

File: src/test/java/seedu/address/model/tag/TagNameContainsCharPredicateTest.java (hunk 1 above)
Comment (HynRidge) [reply src/test/java/seedu/address/model/tag/TagNameContainsCharPredicateTest.java]: Okay fixed!

File: src/test/java/seedu/address/model/tag/TagNameContainsCharPredicateTest.java (hunk 2 above)
Comment (HynRidge) [reply src/test/java/seedu/address/model/tag/TagNameContainsCharPredicateTest.java]: Yep, refactor!

**Review by luo-git — APPROVED** [review]
LGTM

**Pull request merged**